Tasks/JenkinsQueueJob/util.ts

```diff
@@ -133,7 +133,8 @@ function createRootJob(queueUri: string, jobQueue: JobQueue, taskOptions: TaskOp
                 tl.debug(err);
                 defer.resolve(null);
             } else {
-                defer.reject(err);
+                const error = { message: tl.loc('JenkinsJobQueueUriInvalid', JSON.stringify(err)) };
```

Comment:
seems like the tl.debug(err) line should move up to the enclosing if. So, both cases are logged. My guess is that the errors are always logged anyway, but since it is already there.

Reply:
Makes sense, will do.